arch/xtensa/soc/esp32/esp32-mp.c

```diff
@@ -141,6 +173,8 @@ static void appcpu_entry1(void)
  */
 static void appcpu_start(void)
 {
+	smp_log("ESP32: starting APPCPU");
+
 	/* These two calls are wrapped in a "stall_other_cpu" API in
 	 * esp-idf.  But in this context the appcpu is stalled by
```

Comment:
By the way, this was done to work around the DPORT bug (3.10 in the ECO document). We have a better workaround in IDF now, which does not require cross-CPU coordination via interrupts. When reading any DPORT address, if we read any APB address right before reading from DPORT, the issue does not occur. This is implemented here: https://github.com/espressif/esp-idf/blob/64b56beff5a4d6a165bd88027f585bfe51fbb990/components/esp32/dport_access.c#L240-L270
(note that this code needs to be in IRAM so that cache miss does not introduce a delay between APB and DPORT reads).